frontend/libs/studio-components-legacy/src/components/StudioCodelistEditor/StudioCodeListEditor.tsx

```
@@ -119,13 +158,20 @@ function StatefulCodeListEditor({
+      dispatch={dispatch}
     />
   );
 }
 
 type ControlledCodeListEditorProps = Override<
-  Pick<StudioInputTableProps, 'onBlurAny'> & { textResources: TextResource[] },
-  Omit<StatefulCodeListEditorProps, 'onInvalid'>
+  Pick<StudioInputTableProps, 'onBlurAny'> & {
+    textResources: TextResource[];
+    dispatch: Dispatch<ReducerAction>;
+    onCreateTextResource: ({}: CreateTextResourceInternalArgs) => void;
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)